tutorials/detecting-changes-in-sentinel-1-imagery-pt-4/Tutorial_pt_4.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<ul><li>Alphabetize the references</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Not sure what you mean. They are chronological with forenames first.</p>

Reply:
<p>I didn't notice they were first sorted chronologically. This is fine. </p>